### ISSUE TITLE
"atomic diff --json" fails with a traceback

### ISSUE THREAD

**Issue by Jozefinne** [projectatomic/atomic#318]
On Fedora 23 Atomic Host (23.76), atomic-1.8-3.gitcc5997a.fc23.x86_64

When I try to run `atomic diff` with the `--json` option, it fails with a traceback error:

```
-bash-4.3# atomic diff --json centos rhel7
Traceback (most recent call last):
  File "/usr/bin/atomic", line 512, in <module>
    sys.exit(_func())
  File "/usr/lib/python3.4/site-packages/Atomic/diff.py", line 47, in diff
    util.output_json(helpers.json_out)
  File "/usr/lib/python3.4/site-packages/Atomic/util.py", line 94, in output_json
    writeOut(json.dumps(json_data, indent=4, separators=(',', ': ')))
  File "/usr/lib64/python3.4/json/__init__.py", line 237, in dumps
    **kw).encode(obj)
  File "/usr/lib64/python3.4/json/encoder.py", line 194, in encode
    chunks = list(chunks)
  File "/usr/lib64/python3.4/json/encoder.py", line 422, in _iterencode
    yield from _iterencode_dict(o, _current_indent_level)
  File "/usr/lib64/python3.4/json/encoder.py", line 396, in _iterencode_dict
    yield from chunks
  File "/usr/lib64/python3.4/json/encoder.py", line 396, in _iterencode_dict
    yield from chunks
  File "/usr/lib64/python3.4/json/encoder.py", line 317, in _iterencode_list
    yield from chunks
  File "/usr/lib64/python3.4/json/encoder.py", line 429, in _iterencode
    o = _default(o)
  File "/usr/lib64/python3.4/json/encoder.py", line 173, in default
    raise TypeError(repr(o) + " is not JSON serializable")
TypeError: b'/var/run/user/0/systemd/private' is not JSON serializable
```
 Running the command with the other options works OK, only with `--json` it fails on every two images I tried.

**Comment by miabbott** [comment]
I saw the same behavior on F23 Atomic Host, with the version of `atomic` cited.

However, it doesn't appear reproducible using the latest from git master on F23 Cloud.

```
# ./atomic -v
1.9
# git rev-parse HEAD
ce12128cf3acbb3629c8173fe244402ae10a3f80
# ./atomic diff --json rhel7 centos | more                                                                                                                                              
{
    "files_differ": [
        "/bin/tset",
        "/bin/pldd",
        "/bin/sleep",
        "/bin/ptx",
        "/bin/udevadm",
        "/bin/iconv",
        "/bin/base64",
        "/bin/gencat",
        "/bin/pwscore",
        "/bin/tail",
        "/bin/vdir",
        "/bin/lslogins",
        "/bin/systemctl",
        "/bin/pgrep",
        "/bin/lscpu",
...
```

Seems like something was fixed between 1.8 and 1.9

**Comment by rhatdan** [comment]
Fixed in atomic-1.9
